mage_integrations/mage_integrations/destinations/clickhouse/target_clickhouse/README.md

```diff
@@ -0,0 +1,13 @@
+# Clickhouse
```

Comment:
Can you put the README file under this folder? https://github.com/mage-ai/mage-ai/tree/master/mage_integrations/mage_integrations/destinations/clickhouse

Reply:
totally missed that, sorry. Moved in 947b0a65dc6207834bed85c022bd8d8efd608b13